pom.xml

```diff
@@ -1,362 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
I would like to keep this file for sometime. 
Could you just rename this file to `pom_.xml`